addon/components/model-picker.js

```diff
@@ -72,7 +72,9 @@ export default Component.extend(formFieldOptions, {
         list = list.toArray();
       }
 
-      list = list.sort(sortFunction);
+      if (sortFunction) {
```

Comment:
What if there isn't a sortFunction? An array should have a default sort function when passing nothing?